### ISSUE TITLE
Bug: Image alt text isn't used when specified

### ISSUE THREAD

**Issue by thomasheartman** [enonic/app-contentstudio#2946]
## Problem

When specifying an image's alt text via the 'alternative text' field in the content editor, I'd expect that to be used for the `alt` attribute in the result HTML. Instead, the image's display name is used instead.

## Background

During the XP Content 101 course, we were told that you can use an image's display name to set the alt text, but there was also an explicit 'alternative text' attribute that could be set. If they were both set, I (and the instructor) expected the explicit alt text to override the title. However, the image display name was used instead. 

This seems counterintuitive. Is it on purpose or is it a bug? To be clear, I'd expect the alt text to be used if specified. If there is no alt text, use the image display name.

## Related issues?

Seems like it might be tangentially related to [this issue](https://github.com/enonic/app-contentstudio/issues/275)

**Comment by alansemenov** [comment]
Here I have set alt text on the image level (via Content Wizard):
![image](https://user-images.githubusercontent.com/11612490/110470917-beaaf480-80db-11eb-803f-1550d4c84896.png)

Here it gets picked up when I insert the image into an HTML Area ("Insert Image" dialog):
![image](https://user-images.githubusercontent.com/11612490/110471049-ef8b2980-80db-11eb-8ad1-1090fb24a4c2.png)

Here it gets picked up when I insert the image via Image Component on a page:
![image](https://user-images.githubusercontent.com/11612490/110471346-53155700-80dc-11eb-9ac5-bfed139154cd.png)

**Comment by thomasheartman** [comment]
Huh, you're right. I tried this out again now, using the Bootstrap starter kit, but it works as expected. Strange, then, that it didn't work during the content course. May be something else going on there? Anyway, thanks for the input. I'll go ahead and close this now (sorry for the extra work!).
